partcad/src/partcad/test/test.py

```
@@ -0,0 +1,25 @@
+
+from abc import ABC, abstractmethod
+
+from .. import logging as pc_logging
+
+
+class Test(ABC):
+    def __init__(self, name):
+        self.name = name
+
+    @abstractmethod
+    async def test(self, ctx, shape):
```

Ruff (0.8.2)

Missing return type annotation for public function `test`

Add return type annotation: `NoReturn`

(ANN201)